src/Tests/dotnet.Tests/ParserTests/TestParserTests.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.Cli;
+using Xunit;
+using Xunit.Abstractions;
+using System;
+using Parser = Microsoft.DotNet.Cli.Parser;
+
+namespace Microsoft.DotNet.Tests.ParserTests
+{
+    public class TestParserTests : IDisposable
+    {
+        public TestParserTests(ITestOutputHelper output)
+        {
+            this.output = output;
+            this.startingCulture = System.Globalization.CultureInfo.CurrentCulture;
+        }
+
+        private readonly ITestOutputHelper output;
+        private readonly System.Globalization.CultureInfo startingCulture;
+
+        public void Dispose()
+        {
+            System.Globalization.CultureInfo.CurrentCulture = this.startingCulture;
+        }
+
+        [InlineData("en-US")]
+        [InlineData("de-DE")]
+        [Theory]
+        public void TestParserCanGetArgumentFromDoubleDash(string culture)
+        {
+            System.Globalization.CultureInfo.CurrentCulture = new System.Globalization.CultureInfo(culture);
```

Comment:
Tests are run in parallel, and this affects global state, so this may cause flakiness in our tests.

Reply:
Yeah, I know - I couldn't find a way around this aside from reworking a lot of the internals of the CLI parser to use some kind of 'locale provider' abstraction instead. Definitely open to thoughts here.

Reply:
You can go out of proc like runtime tests https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.RemoteExecutor @ViktorHofer is there some guide on how to integrate?

Reply:
Unfortunately we don't have a README but you should just need a PackageReference pointing to the package and then invoke the API.

Reply:
I suppose an alternative here would be to use the existing `DotnetTestCommand` wrappers that actually invoke the CLI on an actual project - but that is _a lot_ of additional work for what should just be a test of the parser.

Reply:
I ended up using out-of-proc test executions against a known, simple test project to validate this.